test/e2e-common.sh

```diff
@@ -255,9 +255,41 @@ function install_ambassador() {
 function install_knative_serving_standard() {
   readonly INSTALL_CERT_MANAGER_YAML="./third_party/cert-manager-${CERT_MANAGER_VERSION}/cert-manager.yaml"
 
-  # If we need to build from source, then kick that off first.
+  echo ">> Installing Knative namespace and CRD"
   if [[ -z "$1" ]]; then
+    # If we need to build from source, then kick that off first.
     build_knative_from_source
+
+    echo "Namespace YAML: ${SERVING_NAMESPACE_YAML}"
+    echo "CRD YAML: ${SERVING_CRD_YAML}"
+    kubectl apply \
+	    -f "${SERVING_NAMESPACE_YAML}" \
+	    -f "${SERVING_CRD_YAML}" || return 1
+    UNINSTALL_LIST+=( "${SERVING_NAMESPACE_YAML}" "${SERVING_CRD_YAML}" )
+  else
+    echo "Knative YAML: ${1}"
+    ko apply -f "${1}" --selector=knative.dev/crd-install=true || return 1
+    cat <<EOF | kubectl apply -f -
```

Comment:
the label is meaningless as-is (that was what I was saying in my thread to myself 😅 ), you should just `kubectl create namespace`.

Reply:
Ah, sorry I misunderstood it. Now, updated.